http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/HttpRoutes.scala

```diff
@@ -99,14 +72,45 @@ class HttpRoutes(
     DebuggingDirectives.logRequestResult(LoggingMagnet(myLoggingFunction))
   }
 
+  implicit val myExceptionHandler = {
+    ExceptionHandler {
+      case leoException: LeoException =>
+        logger.error(s"request failed due to: ${leoException.getMessage}", leoException)
+        complete(leoException.statusCode, leoException.toErrorReport)
+      case e: Throwable =>
+        logger.error(s"Unexpected error occurred processing route: ${e.getMessage}", e)
+        complete(
+          StatusCodes.InternalServerError -> ErrorReport(e.getMessage,
+                                                         Some(StatusCodes.InternalServerError),
+                                                         Seq(),
+                                                         Seq(),
+                                                         Some(e.getClass))
+        )
+    }
+  }
+
+  implicit val myRejectionHandler =
+    RejectionHandler
+      .newBuilder()
+      .handle {
+        case ValidationRejection(msg, cause) =>
+          complete(HttpResponse(StatusCodes.BadRequest, entity = s"${cause.map(_.getMessage).getOrElse(msg)}"))
+      }
+      .handleNotFound(
+        complete(
+          (StatusCodes.NotFound, "API not found. Make sure you're calling the correct endpoint with correct method")
+        )
+      )
+      .result()
+
   val route: Route = {
     logRequestResult {
-      handleExceptions(myExceptionHandler) {
```

Comment:
Does this now use the `myExceptionHandler` and `myRejectionHandler` in implicit scope?

Reply:
yep....this is signature for `seal`
```
  def seal(route: Route)(implicit
    routingSettings: RoutingSettings = null,
                         @deprecated("For binary compatibility. parserSettings is never used", since = "10.1.8") parserSettings:ParserSettings = null,
                         rejectionHandler:                                                                                    RejectionHandler = RejectionHandler.default,
                         exceptionHandler:                                                                                    ExceptionHandler = null)
```